NuGetCleaner/Program.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Reflection;
+using System.IO;
+using CommandLine;
+using System.ComponentModel;
+using NuGet.Configuration;
+
+namespace NuGetCleaner
+{
+    class Program
+    {
+
+        public static void Main(string[] args)
+        {
+            var osNameAndVersion = System.Runtime.InteropServices.RuntimeInformation.OSDescription;
+            if (!osNameAndVersion.Contains("Windows"))
+            {
+                Console.WriteLine("This tool is currently only available for Windows, sorry for the inconvenience!\n");
+                return;
+            }
+
+            Parser.Default.ParseArguments<Options>(args)
+                .WithParsed(options => Execute(options));
+        }
+
+        public static void Execute(Options options)
+        {
+            if (CheckDisableLastAccess() != 2)
+            {
+                Console.Write("\nYour Last Access updates are not currently enabled so this tool will not work. \n" +
+                    "To enable Last Access updates, run powershell as administrator and input:\n\n" +
+                    "\tfsutil behavior set disablelastaccess 2\n\n" +
+                    "You may be asked to reboot for the settings change to take effect\n\n" +
+                    "Note: Last usage time of packages will only be tracked once the setting is enabled.\n" +
+                    "As such, last package usage will only be determined from the enable date onward.\n" +
+                    "For further information view documentation at https://github.com/chgill-MSFT/NuGetCleaner \n");
+                return;
+            }
+
+            var settings = Settings.LoadDefaultSettings(".");
+            var gpfPath = SettingsUtility.GetGlobalPackagesFolder(settings);
+
+            int Days = options.Days;
+
+            if (options.DryRun)
+            {
+                SearchAndPrint(gpfPath, Days);
+            }
+            else {
+                SearchAndDestroy(gpfPath, Days);
+            }
+        }
+
+        public static int CheckDisableLastAccess()
```

Comment:
This method will only work on Windows, making the entire tool Windows only. If that's the intent, ok, but nothing in the package description makes that obvious, only if they look at the readme they might guess. So I can imagine people on Mac or Linux trying to install and use the tool.

You could work cross platform if you did something similar to the following:
1. create an empty file (possibly need to contain at least 1 byte)
2. use `File.SetLastAccessFile` to some time in the past (possibly need to read the value to make sure it's correct)
3. read the file
4. check the last access file, to make sure it's newer than the value you set in 2
5. delete the file